tutorials/math/random_number_generation.rst

```diff
@@ -196,6 +333,54 @@ We can apply similar logic from arrays to dictionaries as well::
         # The same metal may be selected multiple times in succession.
         return random_metal
 
+ .. code-tab:: csharp
```

Comment:
This one should be done another way. IMHO, defining a `struct` and not using a bunch of `static` fields would be better.  
It's kind of a hassle to change this through code suggestions tho.